content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,270 @@
+
+Once the initial connection is established, you can easily enable any of the sub-integrations relevant to your cloud environment. With a single click, Datadog queries AWS with the provisioned IAM role and installs the desired integration. As soon as data returns from the integration, Datadog provisions an out-of-the-box dashboard for that integration, providing immediate and customizable visibility. This guide demonstrates setting up the integration, sending logs from [CloudTrail][2] and the Forwarder Lambda function, and installing the Datadog Agent on an Amazon Linux EC2 instance. See the [Enable sub-integrations section](#enable-sub-integrations) for a list of the available sub-integrations.
+
+This process can be repeated for as many AWS accounts as necessary, or you can also use the [API][3], [AWS CLI][4], or [Terraform][5] to set up multiple accounts at once. For more information, review Datadog's [CloudFormation Guide][6].
+
+## Prerequisites
+
+Before getting started, ensure you have the following prerequisites:
+
+1. An [AWS][7] account. Your AWS user needs the following IAM permissions to successfully run the CloudFormation template:
+
+    * cloudformation:CreateStack
+    * ec2:DescribeSecurityGroups
+    * ec2:DescribeSubnets
+    * ec2:DescribeVpcs
+    * iam:AttachRolePolicy
+    * iam:CreatePolicy
+    * iam:CreateRole
+    * iam:PassRole
+    * iam:PutRolePolicy
+    * iam:UpdateAssumeRolePolicy
+    * kms:Decrypt
+    * lambda:AddPermission
+    * lambda:CreateFunction
+    * lambda:GetCodeSigningConfig
+    * lambda:GetFunction
+    * lambda:GetFunctionCodeSigningConfig
+    * lambda:InvokeFunction
+    * lambda:PutFunctionConcurrency
+    * logs:CreateLogGroup
+    * logs:DescribeLogGroups
+    * logs:PutRetentionPolicy
+    * s3:CreateBucket
+    * s3:GetObject
+    * s3:GetObjectVersion
+    * secretsmanager:CreateSecret
+    * secretsmanager:GetSecretValue
+    * secretsmanager:PutSecretValue
+    * serverless:CreateCloudFormationTemplate
+
+2. A Datadog account and [API key][8]. If you need a Datadog account, [sign up for a free trial][9].
+
+## Setup
+
+1. You must provide a valid Datadog [API Key][10] to the CloudFormation template. You can paste the API key directly into the CloudFront template, or you can store the value in [Secrets Manager][11] and reference the ARN of the secret. First, copy this value to your clipboard.
+
+2. From the AWS tile on the [Integrations page][12] in your Datadog account, select the CloudFormation Template option:
+{{< img src="getting_started/integrations/integration-tile-setup.png" alt="An image from the Datadog AWS integration tile showing the options for establishing the integration. The Role Delegation tab is highlighted.">}}
+
```

Comment:
A step is needed in between 2 & 3 asking the user to change their AWS region in the AWS console if they want to run this template in a different one from us-east-1.